scanpy/_settings.py

```diff
@@ -2,7 +2,22 @@
 import sys
 from pathlib import Path
 from time import time
-from typing import Tuple, Union, Any, List, Iterable, TextIO
+from typing import Tuple, Union, Any, List, Iterable, TextIO, Optional
+
+from . import logging
+from .logging import _set_log_level, _set_log_file, RootLogger
+
+_VERBOSITY_TO_LOGLEVEL = {
+    'error': 'ERROR',
+    'warn': 'WARNING',
```

Comment:
Found one more issue here:

```python
>>> sc.logging.warn("hi")                                                                        
/usr/local/bin/ipython:1: DeprecationWarning: The 'warn' method is deprecated, use 'warning' instead
  #!/usr/local/opt/python/bin/python3.7
WARNING: hi
>>> sc.logging.warning
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-9-8d955ccc14ce> in <module>
----> 1 sc.logging.warning

AttributeError: module 'scanpy.logging' has no attribute 'warning'

Reply:
Good point, I switched everything to `warning`!

Reply:
Was `warn` used before? If so, maybe we could keep a deprecated alias `warn` for backwards compatibility?